packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -1,22 +1,108 @@
 import React, { useRef, useLayoutEffect, useState } from 'react';
 import styled, { css } from 'styled-components';
 import { variables, IconProps, useTheme, Button, Icon, Dropdown } from '@trezor/components';
-import { Translation } from '@suite-components';
-import { useSelector, useActions } from '@suite-hooks';
-import * as routerActions from '@suite-actions/routerActions';
+import { AccountFormCloseButton } from '@suite-components';
+import { isInView } from '../AppNavigationPanel';
+import { useSelector } from '@suite-hooks';
 import { Route } from '@suite-types';
+import AccountStickyContent from '../AccountStickyContent';
+import { MAX_WIDTH, MAX_WIDTH_WALLET_CONTENT } from '@suite-constants/layout';
 
 const { FONT_WEIGHT, FONT_SIZE } = variables;
 
 const SECONDARY_MENU_BUTTON_MARGIN = '12px';
 
-const Wrapper = styled.div<{ settingsWrapper: boolean | undefined }>`
+const Wrapper = styled.div<{ value: boolean; subRoute: boolean | undefined }>`
+    width: 100%;
+    z-index: 2;
     display: flex;
+    background: ${props => props.theme.BG_LIGHT_GREY};
+    justify-content: center;
+
+    ${props =>
+        props.subRoute &&
+        props.value &&
+        `
+            position: sticky;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            height: 0;
+            transform: translate(0, -71px);
+        `}
+
+    ${props =>
+        props.subRoute &&
+        !props.value &&
+        `
+            transition: all 0.3s ease 0s;
+            position: sticky;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            height: 0;
+            transform: translate(0,0);
+        `}
+
+    ${props =>
+        !props.subRoute &&
+        props.value &&
+        `
+
+            position: sticky;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            height: 0;
+            transform: translate(0px,-71px);
+            margin-top: 71px;
+        `}
+
+    ${props =>
+        !props.subRoute &&
+        !props.value &&
+        `
+            transition: all 0.3s ease 0s;
+            position: sticky;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            height: 0;
+            transform: translate(0,0);
+            margin-bottom: 70px;
+        `}
+`;
+
+const InnerWrap = styled.div`
```

Comment:
It looks like it miss some padding
![padding1](https://user-images.githubusercontent.com/3729633/124741774-5ef31780-df1c-11eb-80f9-d826d5e46846.png)
![padding2](https://user-images.githubusercontent.com/3729633/124741788-61557180-df1c-11eb-88e9-a5e9fd2c1b93.png)

Reply:
added padding in 331a45e053a53457e46506ccff18f29dd5f5b5fc

Reply:
now the horizontal padding looks great, thanks, but the vertical padding is somehow broken (the line of active tab is not on the bottom anymore)
![image](https://user-images.githubusercontent.com/3729633/124777368-717f4800-df40-11eb-815c-2c2f8fe49c9d.png)

Reply:
fixed in 8db5d6e6ab43e33341df83c52e00b2e0c733565e